### DISCUSSION TITLE
alternative for make TPL command

### DISCUSSION THREAD

**Discussion by jpdelgad0** [AMReX-Combustion/PeleC#622] [Q&A]
This command is mentioned in the documentation and Readme of PeleC for including third party libraries. I cannot get it to work for me on two different machines and could not find any information for this command unfortunately.

**Comment by esclapez** [comment]
How does it fails ? It indeed requires an internet connection because it will download the SUNDIALs suite.

**Reply by jpdelgad0** [reply]
Thank you. I figured out I was entering the command in the wrong directory excuse me
